plugins/stylelint/rules/no-unknown-token-name.cjs

```diff
@@ -0,0 +1,63 @@
+const { tokens } = require('@rhds/tokens');
+const stylelint = require('stylelint');
+const declarationValueIndex = require('stylelint/lib/utils/declarationValueIndex');
+const parser = require('postcss-value-parser');
+
+const ruleName = 'rhds/no-unknown-token-name';
+
+const messages = stylelint.utils.ruleMessages(ruleName, {
+  expected: 'Expected ...'
+});
+
+const meta = {
+  url: 'https://github.com/RedHat-UX/red-hat-design-tokens/tree/main/plugins/stylelint/rules/no-unknown-token-name.cjs',
+};
+
+/** @type {import('stylelint').Plugin} */
+const ruleFunction = (_, opts, ctx) => {
+  return (root, result) => {
+    const validOptions = stylelint.utils.validateOptions(
+      result,
+      ruleName,
+      {
+        /* .. */
```

Comment:
same as previous comment

Reply:
we can use them later on to add validations. no issue of kb over the wire so might as well keep them